src/screen.js

```diff
@@ -2,51 +2,51 @@
 
 /**
  *  Screen handles calculator screen.  
- *  Constructor.
- *  @param {String} element.
  */
-function Screen(element) {
-    this.element = element;
-    this.reset = false;
-}
-
-/**
- * Allows a new entry. 
- */
-Screen.prototype.resetOnNextInput = function() {
-    this.reset = true;
-}
-
-Screen.prototype.resetOfNextInput = function() {
-    this.reset = false;
-}
+class Screen {
+
+    /**
+     *  Constructor.  
+     *  @param {String} element.
+     */
+    constructor(element) {
+        this.element = element;
+        this.reset = false;
+    }
 
-/**
- * Sets the given number on the screen 
- * @param {Number} number
- */
-Screen.prototype.setNumber = function(number) {
-    this.element.value = number;
-}
+    /**
+     * Allows a new entry. 
+     */
+    resetOnNextInput() {
+        this.reset = true;
+    }
 
-/**
- * Gets the current number from the screen. 
- */
-Screen.prototype.getNumber = function() {
-    return this.element.value;
-};
+    /**
+     * Sets the given number on the screen 
+     * @param {Number} number
+     */
+    setNumber(number) {
+        this.element.value = number;
+    }
 
-/**
- * Adds new number on the screen. 
- * @param {Number} digit.
- */
-Screen.prototype.addDigit = function(digit) {
+    /**
+     * Gets the current number from the screen. 
+     */
+    getNumber() {
+        return this.element.value;
+    }
 
-    if (this.reset === true) {
-        this.element.value = digit;
-        this.reset = false;
-    } else {
-        this.element.value += digit;
+    /**
+     * Adds new number on the screen. 
+     * @param {Number} digit.
+     */
+    addDigit(digit) {
+        if (this.reset === true) {
```

Comment:
If statement accepts boolean, so it is ok to do this:
```
if (a > b) { ... }
if (name === "milos") { ... }
```
since all those conditions are returning either `true` or `false`.

But when you are working with a boolean itself, you do not need to compare it to anything. You can just say:
```
if (this.reset) { ... }